openadapt/config.py

```diff
@@ -139,6 +141,14 @@ def getenv_fallback(var_name: str) -> str:
 DB_URL = f"sqlite:///{DB_FPATH}"
 DIRNAME_PERFORMANCE_PLOTS = "performance"
 
+with open(ROOT_DIRPATH / "pyproject.toml", "r") as f:
```

Comment:
Please add a comment explaining this is a workaround for the lack of support in 3.10 that is in 3.11